client/src/entrypoints/admin/core.js

```diff
@@ -264,12 +264,12 @@ $(() => {
 
     // auto focus on search box
     $input.trigger('focus');
+    const newQuery = $input.val();
```

Comment:
I don't think this should have moved, pulling this out of the search function will likely break the header search. 

This code reads the search field value, which I assume we would only want to do when we call the `search` function.